calcom/cal.com
# chore: add check for booking

## What does this PR do?

- Fixes #XXXX (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)

Everything works fine in handleInstantMeeting and it shouldn't display any toast.

https://calcominc.slack.com/archives/C04CWEU35NV/p1720712414218979

BEFORE:-

![Screenshot 2024-07-11 at 9 30 17 PM](https://github.com/calcom/cal.com/assets/53316345/8d197846-bcd1-4ead-8dff-c38493974523)


## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

1) Create an org team event type
2) Enable instant meeting
3) go to booking page and click on bottom modal "connect now" and fill the form

You shouldn't see any toast error
